CHANGELOG.rst

```diff
@@ -58,11 +65,14 @@ Internal Changes
 - **Gateway:** return 404 if the core service metadata version does not exist instead of redirecting 
   to the endpoint that is using the latest metadata version
   (`#667 <https://github.com/SwissDataScienceCenter/renku-gateway/issues/667>`__)
-  (`2753d07 <https://github.com/SwissDataScienceCenter/renku-gateway/commit/2753d0773e26cb1c74e4be4dd44fe5e77f428657>`__)
+  (`2753d07 <https://github.com/SwissDataScienceCenter/renku-gateway/commit/2753d0773e26cb1c74e4be4dd44fe5e77f428657>`__
+- **UI**: Use a common project URL when invoking renku-core APIs
+  (`#2722 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2722>`_).
 
 Individual components
 ~~~~~~~~~~~~~~~~~~~~~~
 
+- `renku-core 2.6.1 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v2.6.2>`_
```

Comment:
@rokroskar we usually use `renku-python` on release notes
```suggestion
- `renku-python 2.6.2 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/v2.6.2>`_
```

Reply:
oh I didn't change that part - but the version was wrong